src/services/jwtService.js

```diff
@@ -0,0 +1,38 @@
+const jwt = require('jsonwebtoken');
+
+require('dotenv').config();
+
+const sign = (user) => {
+  const token = jwt.sign(user, process.env.JWT_KEY);
```

Comment:
Consider adding an `expiresIn` option to the `jwt.sign` method to set a specific expiration time for the access token. This is a common practice to enhance security.